experimental/ndv4_a100_healthchecks/gen_hpcdiags.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+#Run pre-installed hpc diagnostics and deposit the compressed tar file in OUTDIR
+#
+# takes one argument, the slurm hostname.
+
+USER=cormac
+OUTDIR=/shared/home/${USER}/hpcdiags
```

Comment:
How about creating the output in the directory where the script is called from? This would allow to just execute the script without having to edit it beforehand.